pkg/chain/ethereum/tbtc.go

```diff
@@ -590,13 +924,38 @@ func (mwr *mockWalletRegistry) OnSignatureRequested(
 	})
 }
 
-func (mwr *mockWalletRegistry) GetDKGState() (tbtc.DKGState, error) {
-	mwr.currentDkgMutex.RLock()
-	defer mwr.currentDkgMutex.RUnlock()
+func (tc *TbtcChain) activeWalletPublicKey() ([]byte, bool, error) {
+	walletPublicKeyHash, err := tc.bridge.ActiveWalletPubKeyHash()
+	if err != nil {
+		return nil, false, fmt.Errorf(
+			"cannot get active wallet public key hash: [%v]",
+			err,
+		)
+	}
+
+	if walletPublicKeyHash == [20]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0} {
```

Comment:
I think this could be simplified to:
```suggestion
	if walletPublicKeyHash == [20]byte{} {
```

Reply:
Yep! See d05c5c08